pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -41,7 +41,10 @@ let
     src =
       if stdenv.hostPlatform.system == "x86_64-linux" then
         fetchurl {
-          url = "https://download.nvidia.com/XFree86/Linux-x86_64/${version}/NVIDIA-Linux-x86_64-${version}${pkgSuffix}.run";
+          urls = [
```

Comment:
Since this change affects all versions, did you check if older versions still build? If not please try e.g.
```
nix-build . -A linuxPackages_4_14.nvidia_x11_legacy340
```

Reply:
I would assume that this wouldn't change much since according to https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchurl/default.nix#L41 the original is still tried first.

Anyway `legacy_304` fails due to other reasons.
```
~ took 2s
➜  nix build --no-link nixpkgs.linuxPackages_latest.nvidiaPackages.stable_418 nixpkgs.linuxPackages_latest.nvidiaPackages.legacy_390 nixpkgs.linuxPackages.nvidiaPackages.legacy_340 nixpkgs.linuxPackages_4_14.nvidiaPackages.legacy_304
builder for '/nix/store/wpgja6m5g4y1y0x4g0an3fbsbr7gcyjd-nvidia-graphics-drivers-legacy-304xx_304.137-5.debian.tar.xz.drv' failed with exit code 1; last 10 log lines:
                                   Dload  Upload   Total   Spent    Left  Speed
    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  curl: (22) The requested URL returned error: 404 Not Found

  trying ftp://ftp.funet.fi/pub/mirrors/ftp.debian.org/debian/pool/non-free/n/nvidia-graphics-drivers-legacy-304xx/nvidia-graphics-drivers-legacy-304xx_304.137-5.debian.tar.xz
    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                   Dload  Upload   Total   Spent    Left  Speed
    0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
  curl: (78) RETR response: 550
  error: cannot download nvidia-graphics-drivers-legacy-304xx_304.137-5.debian.tar.xz from any mirror
cannot build derivation '/nix/store/ba09405sk0mib67b0c3x2skzypzmyzl3-nvidia-x11-304.137-4.14.113.drv': 1 dependencies couldn't be built
[4 built (1 failed), 2 copied (152.2 MiB), 67.8 MiB DL]
error: build of '/nix/store/ba09405sk0mib67b0c3x2skzypzmyzl3-nvidia-x11-304.137-4.14.113.drv', '/nix/store/c0qywjagm96in1srh2mbfadhyxmbpdib-nvidia-x11-390.116-5.0.9.drv', '/nix/store/dkyk3iz03qi8wz9dqr3k9i20p9rd7vfs-nvidia-x11-340.107-4.19.36.drv', '/nix/store/qvbmgjjnr3v5zs6yiby5bbs8kaw7wac1-nvidia-x11-418.56-5.0.9.drv' failed

~ took 43s
➜  nix build --no-link nixpkgs.linuxPackages_latest.nvidiaPackages.stable_418 nixpkgs.linuxPackages_latest.nvidiaPackages.legacy_390 nixpkgs.linuxPackages.nvidiaPackages.legacy_340
[3 built]
```